src/HealButton.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+
+export { HealButton as default };
+
+async function healMakerProtocol(debt) {
+  try {
+    console.log(debt);
+    const provider = window.vow.provider;
+    await window.ethereum.enable()
+    const signer = provider.getSigner();
+    const vowWrite = window.vow.connect(signer);
+    const tx = await vowWrite.heal(debt);
+    await provider.waitForTransaction(tx.hash);
+  } catch (error) {
+    console.error(error)
+  }
+}
+
+function HealButton(props) {
+  const [ isHealing, setHealing ] = useState(false);
+  const [ healAmount, setHealAmount ] = useState('');
+
+  useEffect(() => {
+    if (isHealing) {
+      healMakerProtocol(healAmount).then(() => {
+        setHealing(false);
```

Comment:
We should reset `sysDebt` here somehow, but my react foo isn't good enough to do this in 5 minutes.  I would happily take another PR that would set `sysDebt` to 0 or whatever the chain value is after the heal is finished.